tests/default.test.mjs

```diff
@@ -24,7 +26,7 @@ describe('basic functionality', function () {
       '--skip-git',
     ];
 
-    await execa('ember', emberCliArgs, {
+    await execa('npx', emberCliArgs, {
```

Comment:
why did we make this change?

Reply:
not everyone has `ember-cli` installed globally. In order for me to run tests, I needed a way to exec ember-cli

Reply:
PR here: #88